_data/apps.yml

```diff
@@ -1035,3 +1035,12 @@
     - "research"
     - "spreadsheet"
   icon: "dataproofer.png"
+-
+  name: "Electron TimeTable"
```

Comment:
Mind removing the `Electron` prefix from the name and just call it `TimeTable` instead?